client/src/components/History/CurrentHistory/HistoryFilters.vue

```diff
@@ -27,32 +27,49 @@
             </b-input-group-append>
         </b-input-group>
         <div v-if="showAdvanced" class="mt-2" description="advanced filters">
```

Comment:
Can we add the `@keyup.enter="onSearch"` here, instead of adding another child component.

Reply:
Wow how did i not think of that, it works